### PR TITLE
feat: made logo white on dark mode

### PR DESCRIPTION
**Pull request by isaiahthedev**
# Description
This change resolves issue #394 make the logo white on dark mode

## screenshots
Screenshot from 2024-01-15 21-10-04(1)

### CODE REVIEW

**Review by shootermv — APPROVED** [review]
lgtm thanks!

File: src/layout/sidebar/sidebar.scss [hunk 1]
```diff
@@ -52,6 +52,31 @@
...
+    .st0 {
+      fill: none;
+      stroke: white;
+      stroke-width: 23;
+      stroke-miterlimit: 10;
+    }
+    path {
+      stroke-dasharray: 1720;
+      stroke-dashoffset: 1720;
+      animation: dash 0.8s linear forwards;
```
Comment (NoamGaash) [inline src/layout/sidebar/sidebar.scss:68]:
```suggestion
```

File: src/layout/sidebar/sidebar.scss [hunk 2]
```diff
@@ -52,6 +52,31 @@
...
+    }
+    circle {
+      fill: white;
+      stroke-dasharray: 420;
+      stroke-dashoffset: 420;
+      animation: dash 1s linear forwards;
+      animation-delay: 0.5s;
```
Comment (NoamGaash) [inline src/layout/sidebar/sidebar.scss:75]:
```suggestion
```

**Review by NoamGaash — COMMENTED** [review]
looks amazing!
however - have you verified all lines of code are necessary? I believe some lines can be removed

**Comment by isaiahthedev** [comment]
Thanks for the feedback, very helpful since I'm new to git/github. I'm hoping everything is good to go now 💯

**Review by NoamGaash — APPROVED** [review]
great, thanks!

**Pull request merged**